src/processes/follow-topic-process.js

```diff
@@ -52,7 +53,7 @@ const anon_to_sign_user = async (client, user, targetUser) => {
     anon_user_info_emoji_name: anon_init_data.anon_user_info_emoji_name,
     context: 'auto message',
     initiator: user,
-    source_id: ''
+    source_id: sourceId
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `source_id` field has been updated to accept a dynamic value instead of a hardcoded empty string. This change allows for more flexibility and customization when creating new channels. However, ensure that the `sourceId` variable is always defined and sanitized before use to prevent potential security issues such as SQL injection or data corruption.
```diff
    source_id: sourceId
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->